src/views/health-checks.vue

```diff
@@ -46,6 +66,20 @@ export default {
       }
       return currentWindow;
     },
+    checkBlankTargettedLinks() {
+       const riskyAnchors = [...this.getCurrentWindow().document.querySelectorAll('a[target="_blank"]:not([rel*="noopener"])')];
+       for(const anchor of riskyAnchors) {
+           console.log(anchor);
```

Comment:
Is this on purpose?

Reply:
Yes so you could hover over then in DevTools and see them highlighted like this:
![image](https://user-images.githubusercontent.com/17054134/36485213-eaa71ce4-171b-11e8-80c0-0672f0dbd7c8.png)